transport/wseb/src/test/java/org/kaazing/gateway/transport/wseb/specification/wse/connector/OpeningIT.java

```diff
@@ -125,31 +125,6 @@ public void shouldEstablishConnectionWithRequestHeaderXWebSocketProtocol() throw
         k3po.finish();
     }
 
```

Comment:
We should really still test this as it is part of the spec, but to do this you would need to mock the extension factory or something...